### ISSUE TITLE
V 0.6.6 does not show 

### ISSUE THREAD

**Issue by synthview** [jenskutilek/RedArrow-Glyphs#17]
Hello,
I’ve manually updated RA to  Version 0.6.6 on Mavericks OS 10.9.5 G2 Version 2.4.2 (1009) but it doesn’t display in the app :/
```

13/04/17 10:45:54,173 Glyphs[83966]: RedArrow has encountered a fatal error, and will now terminate.
13/04/17 10:45:54,173 Glyphs[83966]: An uncaught exception was raised during execution of the main script:

error: _ModalWindow is overriding existing Objective-C class

This may mean that an unexpected error has occurred, or that you do not have all of the dependencies for this bundle.
13/04/17 10:45:54,173 Glyphs[83966]: Problem with Plugin at Path: /Users/xxxxx/Library/Application Support/Glyphs/Plugins/RedArrow.glyphsReporter
13/04/17 10:45:54,174 Glyphs[83966]: Problem with Plugin at Path: /Users/xxxxx/Library/Application Support/Glyphs/Plugins/RedArrow.glyphsReporter (protocol)
```

**Comment by jenskutilek** [comment]
Looks like a conflict with RoboFab and my class names ... I tried to fix it in v0.6.7, could you try it?

**Comment by MajusCorp** [comment]
Fixed! Thank you.
